cmd/manifest/init-v2.go

```diff
@@ -284,24 +222,26 @@ func (mc *ManifestCommand) deploy(ctx context.Context, opts *InitOpts) error {
 		return err
 	}
 	c := &deploy.DeployCommand{
+		GetDeployer:        deploy.GetDeployer,
 		GetManifest:        mc.getManifest,
 		TempKubeconfigFile: deploy.GetTempKubeConfigFile(mc.manifest.Name),
 		K8sClientProvider:  mc.K8sClientProvider,
-		Builder:            buildv2.NewBuilderFromScratch(mc.analyticsTracker, mc.ioCtrl),
+		Builder:            buildv2.NewBuilderFromScratch(mc.AnalyticsTracker, mc.IoCtrl),
 		GetExternalControl: deploy.NewDeployExternalK8sControl,
 		Fs:                 afero.NewOsFs(),
 		CfgMapHandler:      deploy.NewConfigmapHandler(mc.K8sClientProvider),
 		PipelineCMD:        pc,
 		DeployWaiter:       deploy.NewDeployWaiter(mc.K8sClientProvider),
 		EndpointGetter:     deploy.NewEndpointGetter,
+		IoCtrl:             mc.IoCtrl,
 	}
 
 	err = c.RunDeploy(ctx, &deploy.Options{
 		Name:         mc.manifest.Name,
-		ManifestPath: opts.DevPath,
+		ManifestPath: filepath.Join(opts.Workdir, opts.DevPath),
 		Timeout:      5 * time.Minute,
 		Build:        false,
-		Wait:         false,
+		Wait:         true,
```

Comment:
I've changed `Wait` to `true` because it's important that the services are deployed in case the user wants to use `--configure-devs`, also, if we consider it's the first time they deploy these services, I think it makes sense to wait and stream the logs to the user